docs/swarm/swarm-tutorial/add-nodes.md

```diff
@@ -23,24 +23,35 @@ This tutorial uses the name `worker1`.
 [Create a swarm](create-swarm.md) tutorial step to create a worker node joined to the existing swarm:
 
     ```bash
-    $ docker swarm join --secret 4ao565v9jsuogtq5t8s379ulb \
-      --ca-hash sha256:07ce22bd1a7619f2adc0d63bd110479a170e7c4e69df05b67a1aa2705c88ef09 \
-      192.168.99.100:2377
+    $ docker swarm join \
+    --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+    192.168.99.100:2377
+
+    This node joined a swarm as a worker.
     ```
 
     If you don't have the command available, you can run the following command:
 
     ```bash
-    docker swarm join --secret <SECRET> <MANAGER-IP>:<PORT>
+    docker swarm join --token <WORKER-TOKEN> <MANAGER-IP>:<PORT>
+    ```
+
+    To find the worker token, run the following command on the manager node:
+
+    ```bash
+    $ docker swarm join-token worker
+
+    SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c
```

Comment:
Not related to this PR, but looking at this, it would be nice to have `docker swarm join token --something` that prints the usage instructions (like on `docker swarm init`)

Perhaps even the reverse; print it by default, unless a `-q` / `--quiet` is passed